### PR TITLE
Add BabelMap 12.1.0.1

### PR DESCRIPTION
**Pull request by boltomli**
A free character map application that allows you to browse through the entire Unicode character repertoire.

### CODE REVIEW

File: bucket/babelmap.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "http://www.babelstone.co.uk/Software/BabelMap.html",
+    "description": "A free character map application to browse through the entire Unicode character repertoire.",
+    "version": "12.1.0.1",
+    "url": "http://www.babelstone.co.uk/Software/Download/BabelMap.zip",
+    "hash": "3285b9c237db93f67c78a3c7e81e2f623345c54e4d44b8f6919e2d8913875527",
+    "bin": [
```
Comment (Ash258) [inline bucket/babelmap.json]:
```suggestion
    "bin": "BabelMap.exe",
```

File: bucket/babelmap.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    ],
+    "shortcuts": [
+        [
+            "BabelMap.exe",
+            "BabelMap"
+        ]
+    ],
+    "checkver": {
+        "url": "http://www.babelstone.co.uk/Software/BabelMap_Versions.html",
+        "re": "([\\d.]+\\d+)"
```
Comment (Ash258) [inline bucket/babelmap.json]:
```suggestion
        "regex": ">([\\d.]+)<"
```

File: bucket/babelmap.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    }
```
Comment (Ash258) [inline bucket/babelmap.json]:
```suggestion
    },
    "autoupdate": {
        "url": "http://www.babelstone.co.uk/Software/Download/BabelMap.zip"
    }
```

**Comment by rasa** [comment]
See also https://github.com/rasa/scoops/blob/master/babelmap.json and https://github.com/rasa/scoops/blob/master/babelpad.json

**Comment by linsui** [comment]
@boltomli Please see the [docs](https://github.com/lukesampson/scoop/wiki/App-Manifests)

> However if you declare just one shim like this, you must ensure it's enclosed in an outer array, e.g: "bin": [ [ "program.exe", "alias" ] ]. Otherwise it will be read as separate shims.

and [rasa's manifest](https://github.com/rasa/scoops/blob/79e730254ebf1ff2ff6be1e6cbc937ce1c79e87a/babelmap.json#L11-L16)

```json
    "bin": "BabelMap.exe",
```

```json
    "shortcuts": [
        [
            "BabelMap.exe",
            "BabelMap - Unicode character map viewer"
        ]
    ],
```

File: bucket/babelmap.json (hunk 1 above)
Comment (Ash258) [reply bucket/babelmap.json]: Why you skipped this?

**Comment by boltomli** [comment]
Sorry I'll come back to this one later, thanks all for the review!

File: bucket/babelmap.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        ]
+    ],
+    "checkver": {
+        "url": "http://www.babelstone.co.uk/Software/BabelMap_Versions.html",
+        "regex": ">([\\d.]+)<"
+    },
+    "autoupdate": {
+        "url": "http://www.babelstone.co.uk/Software/Download/BabelMap.zip"
+    },
+    "license": "Freeware"
```
Comment (Ash258) [inline bucket/babelmap.json]:
```suggestion
```

File: bucket/babelmap.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+    "homepage": "http://www.babelstone.co.uk/Software/BabelMap.html",
+    "description": "A free character map application to browse through the entire Unicode character repertoire.",
+    "version": "12.1.0.1",
```
Comment (Ash258) [inline bucket/babelmap.json]:
```
    "version": "12.1.0.1",
    "license": "Freeware",
```

**Pull request merged**